pkg/cmd/pipelineresource/list.go

```diff
@@ -17,12 +17,12 @@ package pipelineresource
 import (
 	"fmt"
 	"os"
-	"sort"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/spf13/cobra"
 	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/pipelineresource/sort"
```

Comment:
Run `gofmt -w` on this file and push up resulting change to resolve the build failure.